doc/source/whatsnew/v1.0.0.rst

```diff
@@ -858,6 +858,7 @@ Other
 - Bug in :meth:`DataFrame.append` that raised ``IndexError`` when appending with empty list (:issue:`28769`)
 - Fix :class:`AbstractHolidayCalendar` to return correct results for
   years after 2030 (now goes up to 2200) (:issue:`27790`)
+- Fixed :class:`IntegerArray` returning ``NA`` rather than ``inf`` for operations dividing by 0 (:issue:`27398`)
```

Comment:
This is a bit confusingly worded I think, as I would interpret this that it now returns NaN and no longer inf, while it is the other way around? 

Also NA -> NaN, as the result is about floats with NaNs

Reply:
for non-trivially to understand changes we usually have a sub-section, but up to you.